.sourcery_automockable_config.yml

```diff
@@ -0,0 +1,8 @@
+sources:
```

Comment:
ignore this for now as i'm still trying to get this to work, the main issue was cocoapods and my previews weren't linking

Reply:
*used the binary release and integrated directly into the project